evergreen.yml

```diff
@@ -60,3 +60,14 @@ tasks:
           . venv/bin/activate
 
           pydocstyle src
+
+- name: check_isort
```

Comment:
We might want to consider using the [pytest plugin ](https://pypi.org/project/pytest-isort/)